snuba/cli/subscriptions.py

```diff
@@ -0,0 +1,197 @@
+import signal
+import uuid
+from concurrent.futures import ThreadPoolExecutor
+from contextlib import closing
+from datetime import timedelta
+from typing import Optional, Sequence
+
+import click
```

Comment:
is this intentionally separated from the other module imports (signal, uuid) ?

Reply:
Yes, this is as dictated by PEP 8:

> Imports should be grouped in the following order:
>
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.